golang-project/wheezy.bash

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+cat << EOF
+*******************************************************************************
+*** ERROR: golang-project:wheezy is deprecated, use the go builder instead. ***
```

Comment:
refer to the whole image names w/ gcr.io

Reply:
Done

Reply:
Done